### DISCUSSION TITLE
Invitation to submit a paper to NLP-OSS workshop

### DISCUSSION THREAD

**Discussion by alvations** [zilliztech/GPTCache#422] [General]
Dear GPTCache developers,

The work you've done in the GPTCache is really interesting! And it highlights an important synergy aspect on cross-software collaborations which is rare in machine learning and natural language processing frameworks, (langchain + minigpt4 + llama) 

We would like to invite you to submit a paper to the NLP-OSS workshop describing GPTCache and possibly how/why the tool was created and if you have any other knowledge/demonstration of the tool particular to NLP. It'll be good to share with the NLP academic research community that will participate in the workshop. 

NLP-OSS is a little different from a typical academic workshop and it's a venue to share open source software (OSS) work like what you have done in GPTCache. The focus will be less on how much scientific breakthrough in typical NLP metrics and more on the engineering and community aspect of NLP, e.g. how/why the OSS was created and/or how the users/community using the OSS contribute to the tool improvements. 

The workshop would be co-located in EMNLP 2023 in Singapore, December 2023. https://nlposs.github.io/

We look forward to your submission to the NLP-OSS workshop!

Best Regards,
Liling
On behalf of NLP-OSS organizers

**Comment by SimFG** [comment]
@alvations Thanks your invitation. Through the introduction (<https://nlposs.github.io/>), we understand the content of the submission and its related date. Besides these, are there any additional materials and considerations required?
